plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHivePartitionsOnDataLake.java

```diff
@@ -32,17 +39,18 @@
 import static java.util.stream.Collectors.joining;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-public abstract class BaseTestHiveInsertOverwrite
+public abstract class BaseTestHivePartitionsOnDataLake
```

Comment:
The "Partitions" in class name is wrong, as the tests cover both partitioned and non-partitioned tables.

Reply:
Hmm in general all of those test are about partitions in negative and positive way.
Tests that do not use partitioned tables test if we deny using feature reserved for partitioned tables in context of non partitioned. So i think this is fair to name it Partitions test.

Reply:
I am also not a big fan of this name.
Maybe `BaseTestHiveWithRealMetastoreAndMinio`?

Reply:
Renamed to `BaseTestHiveOnDataLake`